opm/simulators/flow/KeywordValidation.cpp

```diff
@@ -0,0 +1,153 @@
+/*
+  Copyright 2021 Equinor.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <map>
+#include <string>
+#include <type_traits>
+
+#include <fmt/format.h>
+
+#include <opm/common/utility/OpmInputError.hpp>
+#include <opm/parser/eclipse/Parser/ErrorGuard.hpp>
+#include <opm/parser/eclipse/Parser/ParseContext.hpp>
+#include <opm/simulators/flow/KeywordValidation.hpp>
+
+namespace Opm
+{
+
+namespace KeywordValidation
+{
+
+    std::string get_error_report(const std::vector<ValidationError>& errors, const bool critical)
+    {
+        const std::string keyword_format = "  {keyword}: keyword not supported\n";
+        const std::string option_format = "  {{keyword}}: invalid value '{}' for option {}\n";
+        const std::string location_format = "  In file: {file}, line {line}\n";
+
+        std::string report;
+        for (const ValidationError& err : errors) {
+            if (err.critical == critical) {
+                if (err.item_name && err.item_value) {
+                    std::string message = fmt::format(option_format, *(err.item_value), *(err.item_name));
+                    report.append(OpmInputError::format(message, err.location));
+                } else {
+                    report.append(OpmInputError::format(keyword_format, err.location));
+                }
+                report.append(OpmInputError::format(location_format, err.location));
+                if (err.user_message) {
+                    report.append("  NOTE: " + *(err.user_message) + "\n");
```

Comment:
Please remove 'NOTE: ' from the message, so that it reads:
`report.append("  " + *(err.user_message) + "\n");`